app/helpers/sessions_helper.rb

```diff
@@ -9,31 +9,44 @@ def remember user
     cookies.permanent[:remember_token] = user.remember_token
   end
 
-  def forget user
-    user.forget
-    cookies.delete :user_id
-    cookies.delete :remember_token
-  end
-
   def current_user
-    if session[:user_id]
-      @current_user ||= User.find_by id: session[:user_id]
-    elsif cookies.signed[:user_id]
-      user = User.find_by id: cookies.signed[:user_id]
+    if user_id = session[:user_id]
```

Comment:
sao lại dùng phép gán ở đây mà k phải là toán tử so sánh

Reply:
Dùng gán để vừa kiểm tra vừa gán giá trị ạ còn so sánh chỉ kiểm tra thôi ạ